setup.cmd

```diff
@@ -0,0 +1,15 @@
+@echo off
```

Comment:
Does `@echo off` potentially obscure some useful information like "what command is called"?

Reply:
At least not for me